mmv1/products/workstations/WorkstationConfig.yaml

```diff
@@ -136,25 +136,43 @@ properties:
   - !ruby/object:Api::Type::Time
     name: "createTime"
     description: |
-      Time the Instance was created in UTC.
+      Time when this resource was created.
     output: true
+  - !ruby/object:Api::Type::String
+    name: "idleTimeout"
+    default_from_api: true
+    description: |
+      How long to wait before automatically stopping an instance that hasn't received any user traffic. A value of 0 indicates that this instance should never time out due to idleness. Defaults to 20 minutes.
+      A duration in seconds with up to nine fractional digits, ending with 's'. Example: "3.5s".
+  - !ruby/object:Api::Type::String
+    name: "runningTimeout"
+    default_from_api: true
+    description: |
+      How long to wait before automatically stopping a workstation after it started. A value of 0 indicates that workstations using this configuration should never time out. Must be greater than 0 and less than 24 hours if encryptionKey is set. Defaults to 12 hours.
+      A duration in seconds with up to nine fractional digits, ending with 's'. Example: "3.5s".
   - !ruby/object:Api::Type::NestedObject
     name: "host"
     description: |
       Runtime host for a workstation.
-    immutable: true
     default_from_api: true
+    update_mask_fields:
+      - "host.gceInstance.machineType"
+      - "host.gceInstance.poolSize"
+      - "host.gceInstance.tags"
+      - "host.gceInstance.disablePublicIpAddresses"
+      - "host.gceInstance.shieldedInstanceConfig.enableSecureBoot"
+      - "host.gceInstance.shieldedInstanceConfig.enableVtpm"
+      - "host.gceInstance.shieldedInstanceConfig.enableIntegrityMonitoring"
+      - "host.gceInstance.confidentialInstanceConfig.enableConfidentialCompute"
     properties:
       - !ruby/object:Api::Type::NestedObject
         name: "gceInstance"
```

Comment:
Can we also includes other fields changed to updatable in the update tests if possible? Looks like the test case only covers `machine_type` now.

Reply:
Changed the test to update all of the fields.